example/bottle/server.py

```diff
@@ -26,7 +29,9 @@ def handle_zero_div():
     except Exception as e:
```

Comment:
(GitHub won't let me comment one line higher)

flake8 on this file is complaining about `x` being an unused local variable, and this seems to be the only lint issue left on this file. Perhaps this should be fixed and then `example/bottle` can be added to scripts/lint.sh.

Reply:
I've raised an internal ticket to address this - suggest not doing it here as it affects the other examples too.